site/gatsby-site/src/components/checklists/CheckListForm.js

```diff
@@ -174,17 +174,19 @@ export default function CheckListForm({
   };
 
   const updateRisk = (risk, attributeValueMap) => {
-    const updatedRisks = [...values.risks];
+    const updatedRisks = JSON.parse(JSON.stringify(values.risks));
```

Comment:
Be careful that this cloning method results in data loss, dates for example get turned into strings.

`lodash.cloneDeep` is better.

Reply:
Thanks for the tip!